vscgallery is not behaving properly

The usual display of files is not available on latest build.

Client receives:

```
This XML file does not appear to have any style information associated with it. The document tree is shown below.
<error>
<title>500 Internal Server Error</title>
</error>
```

Logs on the container show

```
vscsync-1     | /opt/vscoffline/sync.py:358: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
vscsync-1     |   'updated': datetime.datetime.utcnow()
vscsync-1     | [I 250108 01:25:53 sync:819] Going to sleep for 12:00:00
vscgallery-1  | [2025-01-08 01:26:59 +0000] [1] [INFO] Starting gunicorn 23.0.0
vscgallery-1  | [2025-01-08 01:26:59 +0000] [1] [INFO] Listening at: https://0.0.0.0:9443 (1)                                                                               
vscgallery-1  | [2025-01-08 01:26:59 +0000] [1] [INFO] Using worker: gthread
vscgallery-1  | [2025-01-08 01:26:59 +0000] [2] [INFO] Booting worker with pid: 2
vscgallery-1  | [2025-01-08 01:27:08 +0000] [2] [WARNING] Invalid request from ip=10.89.1.1: [SSL: SSLV3_ALERT_CERTIFICATE_UNKNOWN] ssl/tls alert certificate unknown (_ssl.c:2570)
vscgallery-1  | [2025-01-08 01:27:08 +0000] [2] [WARNING] Invalid request from ip=10.89.1.1: [SSL: SSLV3_ALERT_CERTIFICATE_UNKNOWN] ssl/tls alert certificate unknown (_ssl.c:2570)
vscgallery-1  | [2025-01-08 01:27:08 +0000] [2] [WARNING] Invalid request from ip=10.89.1.1: [SSL: SSLV3_ALERT_CERTIFICATE_UNKNOWN] ssl/tls alert certificate unknown (_ssl.c:2570)
vscgallery-1  | 2025-01-08 01:27:08 [FALCON] [ERROR] GET / => Traceback (most recent call last):
vscgallery-1  |   File "falcon/app.py", line 440, in falcon.app.App.__call__
vscgallery-1  |   File "/opt/vscoffline/server.py", line 356, in on_get
vscgallery-1  |     resp.body = f.read()
vscgallery-1  |     ^^^^^^^^^
vscgallery-1  |   File "falcon/response.py", line 221, in falcon.response.Response.body                                                                                     
vscgallery-1  | falcon.util.deprecation.AttributeRemovedError: The body attribute is no longer supported. Please use the text attribute instead.
vscgallery-1  | 
vscgallery-1  | 10.89.1.1 - - [08/Jan/2025:01:27:08 +0000] "GET / HTTP/1.1" 500 93 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 Edg/131.0.0.0"
```

--- a/vscoffline/server.py
+++ b/vscoffline/server.py
@@ -87,7 +87,7 @@ class VSCRecommendations(object):
         resp.status = falcon.HTTP_200
         resp.content_type = 'application/octet-stream'
         with open(vsc.ARTIFACT_RECOMMENDATION, 'r') as f:
-            resp.body = f.read()
+            resp.text = f.read()
 
 class VSCMalicious(object):
 
@@ -98,7 +98,7 @@ class VSCMalicious(object):
         resp.status = falcon.HTTP_200
         resp.content_type = 'application/octet-stream'
         with open(vsc.ARTIFACT_MALICIOUS, 'r') as f:
-            resp.body = f.read()
+            resp.text = f.read()
 
 class VSCGallery(object):
 
@@ -353,7 +353,7 @@ class VSCIndex(object):
     def on_get(self, req, resp):
         resp.content_type = 'text/html'
         with open('/opt/vscoffline/vscgallery/content/index.html', 'r') as f:
-            resp.body = f.read()
+            resp.text = f.read()
         resp.status = falcon.HTTP_200
 
 class VSCDirectoryBrowse(object):
@@ -370,9 +370,9 @@ class VSCDirectoryBrowse(object):
         resp.content_type = 'text/html'
         # Load template and replace variables
         with open('/opt/vscoffline/vscgallery/content/browse.html', 'r') as f:
-            resp.body = f.read()
-        resp.body = resp.body.replace('{PATH}', requested_path)
-        resp.body = resp.body.replace('{CONTENT}', self.simple_dir_browse_response(requested_path))
+            resp.text = f.read()
+        resp.text = resp.text.replace('{PATH}', requested_path)
+        resp.text = resp.text.replace('{CONTENT}', self.simple_dir_browse_response(requested_path))
         resp.status = falcon.HTTP_200
 
     def simple_dir_browse_response(self, path):

